pkg/controller/common/util.go

```diff
@@ -701,6 +702,42 @@ func ValidateClone(sourcePVC *v1.PersistentVolumeClaim, spec *cdiv1.DataVolumeSp
 	return nil
 }
 
+// ValidateSnapshotClone compares a snapshot clone spec against its source snapshot to validate its creation
+func ValidateSnapshotClone(sourceSnapshot *snapshotv1.VolumeSnapshot, spec *cdiv1.DataVolumeSpec) error {
```

Comment:
Curious why you put this here if it's only used in the webhook?

Reply:
I had to use it in the controller itself and missed that part